build/dependencies.props

```diff
@@ -27,6 +27,7 @@
     <MicrosoftVisualStudioShellInterop80PackageVersion>8.0.50727</MicrosoftVisualStudioShellInterop80PackageVersion>
     <MicrosoftVisualStudioShellInterop90PackageVersion>9.0.30729</MicrosoftVisualStudioShellInterop90PackageVersion>
     <MicrosoftVisualStudioShellInteropPackageVersion>7.10.6071</MicrosoftVisualStudioShellInteropPackageVersion>
+    <MicrosoftVisualStudioTextUIPackageVersion>15.6.161-preview</MicrosoftVisualStudioTextUIPackageVersion>
```

Comment:
This already exists in Universe with this version so did not need to do a PR there.

Reply:
Are there other VS dependencies that should be versioned along with this? We really want to try and keep a consistent set of deps if possible.

Reply:
Yes, we only list top level dependencies in the Razor repo but Universe has the entire graph (all versioned consistently).